example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js

```diff
@@ -88,6 +89,15 @@ describe('The FTP Ingest Granules workflow', () => {
     });
 
     it('makes the granule available through the Cumulus API', async () => {
+      // This assertion is to check that the granule has been updated in dynamo
+      // before performing further checks
+      const record = await waitForModelStatus(
```

Comment:
I think we might want to move this into the `beforeAll` before it requests the granule response from the API? Because it uses that response in subsequent tests in this file

https://github.com/nasa/cumulus/pull/1134/files#diff-f7ba682f98b85974d5ba75ce744e281fR72

Reply:
yeah you're right. my b. going to update.